dev/com.ibm.ws.security.mp.jwt.1.1/resources/com/ibm/ws/security/mp/jwt/v11/resources/MicroProfileJwtMessages.nlsprops

```diff
@@ -0,0 +1,31 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.security/resources/com/ibm/ws/security/mp/jwt/v11/resources/MicroProfileJwtMessages.nlsprops
+#COMPONENTPREFIX CWWKS
+#COMPONENTNAMEFOR WebSphere Application Server MicroProfile JWT 1.1
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+# Message prefix block: CWWKS5750 - CWWKS5799
+
+MPJWT_11_CONFIG_PROCESSED=CWWKS5750I: The MicroProfile JWT v11 configuration [{0}] was successfully processed.
+MPJWT_11_CONFIG_PROCESSED.explanation=The indicated configuration was successfully processed.
```

Comment:
A style edit here with regards to verb tense: 
MPJWT_11_CONFIG_PROCESSED=CWWKS5750I: The MicroProfile JWT v11 configuration [{0}] processed successfully.

MPJWT_11_CONFIG_PROCESSED.explanation=The indicated configuration processed successfully.

Reply:
I'll update your recommendation by using another PR since additional message modifications are coming over the weekend. Thank you for your review.